cmd/podman/system/unshare.go

```diff
@@ -14,7 +14,7 @@ import (
 var (
 	unshareDescription = "Runs a command in a modified user namespace."
 	unshareCommand     = &cobra.Command{
-		Use:                   "unshare [COMMAND [ARG ...]]",
+		Use:                   "unshare [COMMAND] [ARG...]",
```

Comment:
This is not really accurate: a naïve reader could easily interpret that as "COMMAND -or- ARG" (which is of course meaningless). If at all possible, I'd like to see the original usage preserved.